docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -291,10 +291,115 @@ baseline. We therefore conclude that this version of CometBFT has passed the tes
+| :------------------------------------------------------------------: | :----------------------------------------------------------------: |
+| ![rotating-heights-ephe-bl](img38/rotating/rotating_eph_heights.png) | ![rotating-heights-ephe](imgs/v1/rotating/metrics/rotating_eph_heights.png) |
+
+In both cases, we see the main loop of the `rotating` test case repeat itself a number of times.
+Ephemeral nodes are stopped, their persisted state is wiped out, their config is transferred over
+from the orchestrating node, they are started, we wait for all of them to catch up via blocksync,
+and the whole cycle starts over. All these steps are carried out via `ansible` playbooks.
+
+We see that there are less cycles in `v1.0.0`. The reason is the following.
+All `ansible` steps are currently run from the orchestrating node (i.e., the engineer's laptop).
+The orchestrating node when running the rotating nodes test case for `v1.0.0`
+was connected to a network connection that was slower than when the equivalent test was run for `v0.38.x`.
+This caused the steps reconfiguring ephemeral nodes at the end of each cycle to be somewhat slower.
+This can be noticed in the graphs when comparing the width (in x-axis terms) of the gaps without metric
+from the end of a cycle to the beginning of the next one.
+
+If we focus on the _width_ of periods when ephemeral nodes are blocksynching, we see their are slightly narrower
+in `v1.0.0`. This is likely due to the improvements introduced as part of the following issues
+[#1283](https://github.com/cometbft/cometbft/issues/1283),
+[#2379](https://github.com/cometbft/cometbft/issues/2379), and
+[#2465](https://github.com/cometbft/cometbft/issues/2465).
+
+The following plots show the transactions processed per minute.
+
+|                            v0.38.0                             |                          v1.0.0 (without LE)                          |
+| :------------------------------------------------------------: | :-------------------------------------------------------------------: |
+| ![rotating-total-txs-bl](img38/rotating/rotating_txs_rate.png) | ![rotating-total-txs](imgs/v1/rotating/metrics/rotating_txs_rate.png) |
+
+They seem similar, except for an outlier in the `v1.0.0` plot.
+
+#### Peers
+
+The plots below show the evolution of the number of peers throughout the experiment.
+
+|                         v0.38.0                         |                      v1.0.0 (without LE)                       |
+| :-----------------------------------------------------: | :------------------------------------------------------------: |
+| ![rotating-peers-bl](img38/rotating/rotating_peers.png) | ![rotating-peers](imgs/v1/rotating/metrics/rotating_peers.png) |
+
+The plotted values and their evolution show the same dynamics in both plots.
+Nevertheless, all nodes seem to acquire more peers when ephemeral node are catching up in the `v1.0.0` experiment.
```

Comment:
Is there an explanation or hypothesis for this?

Reply:
I don't have a clear explanation of hypothesis, as we didn't really touch the p2p layer significantly between `v0.38.x` and `v1.x`. Maybe the improvements @melekes did to blocksync ?

In any case, it represents an improvement on `v1.x` w.r.t `v0.38.x`.

@cason do you have an idea what it could be?